contrib/vagrant/start.sh

```diff
@@ -335,6 +335,7 @@ function write_cilium_cfg() {
     fi
 
     cilium_options+=" ${TUNNEL_MODE_STRING}"
+    cilium_options+=" --device enp0s8"
```

Comment:
Why this change? the `device` option has been deprecated.

Reply:
Ah, It should be `--devices`..